index.html

```diff
@@ -14,30 +14,266 @@
+                      </div>
+                      <div class="buttom-container">
+                        <input type="button" value="See Project" class="project-buttom" />
+                      </div>
+                    </div>
+                </div>
+            </div>
+            <div class="sample-project project-4">
+                <div class="project-header">
+                    <div class="image-holder bg">
+                        <img src="./Assets/SnapshootPortfolio3.png" alt="header" class="header-img"/>
+                    </div>
+
+                </div>
+                <div class="project-content">
+                    <div class="project-description">
+                      <div class="title-container">
+                        <h4 class="description-title">Multi-Post Stories</h4>
+                      </div>
+                      <div class="subtitle-container">
+                        <ul class="subtitle-items">
+                          <li class="list-less"><p>CANOPY</p></li>
+                          <li class="list"><p>Back End Dev</p></li>
+                          <li class="list"><p>2015</p></li>
+                        </ul>
+                      </div>
+                      <div class="article-container">
+                        <p class="description-item">
+                          A daily selection of privately perosonalized reads; no accounts
+                          or sign-ups required.
+                        </p>
+                      </div>
+                      <div class="tag-container">
+                        <ul class="tag-item">
+                          <li class="tag-list-item"><a class="tag-btn" href="#">html</a></li>
+                          <li class="tag-list-item"><a class="tag-btn" href="#">css</a></li>
+                          <li class="tag-list-item"><a class="tag-btn" href="#">javaScript</a></li>
+                        </ul>
+                      </div>
+                      <div class="buttom-container">
+                        <input type="button" value="See Project" class="project-buttom" />
+                      </div>
+                    </div>
+                </div>
+            </div>            
+        </section>
+
+        <section id="about-myself">
+            <div class="about-content">
+                <div class="about-1">
+                    <div class="about-title-box">
+                        <h2 class="about-header">About myself</h2>
+                    </div>
+                    <div class="about-desc-box">
+                        <p class="description-item">I'm a software developer! I can help you build a product, feature or website look througth some of my work experience! if you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+                    </div>
+                    <div class="Right contact">
+                        <p class="media-text">LET’S CONNECT</p>
```

Comment:
- [ ] Kindly use a header tag hx instead of a p tag.
- [ ] Also for the about section kindly use flexbox not absolute values to align it.
- [ ] Try to match the design of the about section with the one in Figma by removing extra white space in the design and preventing it from being scrolled to the right.